config/routes.rb

```diff
@@ -59,6 +59,7 @@
 
   resource :preferences, only: %i[edit update], as: :global_preferences
   resource :messageboard, path: 'messageboards', only: [:new]
+  get '/messageboard_groups/:id', action: :show, controller: 'messageboard_groups', as: :show_messageboard_group
```

Comment:
1. Change to "messageboard-groups" (Thredded uses hyphens throughout).
2. "messageboard-groups" needs to be added to the list at https://github.com/thredded/thredded/blob/13f4861b978a1450a0852a11f98582b4fb7ebac0/app/models/thredded/messageboard.rb#L9